frontend/src/App.js

```diff
@@ -1,9 +1,96 @@
-import React from 'react'
+import React, { useState } from "react";
+
+import { SignUp } from "./components/SignUp";
+import { SignIn } from "./components/SignIn";
+import { Secret } from "./components/Secret";
+
+import { signUpUrl, signInUrl } from "./paths/api-paths";
+import { ToggleButton } from "./components/ToggleButton";
+import { SignOutButton } from "./components/SignOutButton";
 
 export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+	const [mode, setMode] = useState("signIn");
+	const tokenFromStorage = () => window.localStorage.getItem("tokenAuth") || "";
+	const [token, setToken] = useState(tokenFromStorage);
+	const [signUpOk, setSignUpOk] = useState();
+	const [signInOk, setSignInOk] = useState();
+
+	const handleChangeMode = (modeFromButton) => {
+		setMode(modeFromButton);
+	};
+
+	const handleSignOut = () => {
+		setToken("");
+		//remove token from local storage
+		localStorage.clear();
```

Comment:
It is better to remove only a single field from the local storage here. In a bigger app, you can have more data stored in like bright/dark mode or bigger letters mode when someone needs them. 🌈